nextjs-logs-manager/app/pages/logs/hooks/useLogsManager.ts

```
@@ -0,0 +1,193 @@
+      const newSortedLogs = sortLogsByTimestamp(logs, newDirection)
+      setLogs(newSortedLogs)
+      setFilteredLogs(sortLogsByTimestamp(filteredLogs, newDirection))
+      return newDirection
+    })
+  }
+
+  // Obtener valores únicos de los logs
+  const getUniqueValues = (logs: LogEntry[]) => {
+    const services = logs
+      .reduce((acc, log) => 
+        acc.includes(log.service) ? acc : [...acc, log.service], 
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)